app/api/agent-templates/route.ts

```diff
@@ -0,0 +1,24 @@
+import { NextResponse } from "next/server";
+import supabase from "@/lib/supabase/serverClient";
+
+export async function GET() {
+  try {
+    const { data, error } = await supabase
+      .from('agent_templates')
+      .select('*')
+      .order('title');
```

Comment:
put all supabase queries in `/lib/supabase`
- actual: `app/api/agent-templates/route.ts`
- required: `lib/supabase/agent_templates/getAgentTemplates.ts`